### PR TITLE
Some bug fixes

### LINKED ISSUE #2179: Bug Report About Bootstrap-table Filter control extension
I add the 'Clear Filters' button,and when I click the button,there is an error in the console.

image

### PR DESCRIPTION
**Pull request by djhvscf**
Example: https://jsfiddle.net/djhvscf/fp25opbb/1/

### CODE REVIEW

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -1956,7 +1956,7 @@
         }
         request = $.extend({}, calculateObjectValue(null, this.options.ajaxOptions), {
             type: this.options.method,
-            url: this.options.url,
+            url: (url === undefined ? this.options.url : url),
```
Comment (wenzhixin) [inline src/bootstrap-table.js]: url || this.options.url

Comment (djhvscf) [inline src/bootstrap-table.js]: :facepunch: yes, you're right.. :o: 

Let me change it

File: src/bootstrap-table.css [hunk 2]
```diff
@@ -196,7 +196,7 @@
     line-height: 1.428571429;
 }
 
-.fixed-table-toolbar .bars,
+.fixed-table-toolbar .bars-bs-table,
```
Comment (wenzhixin) [inline src/bootstrap-table.css]: I think here is better to use `.bs-bars`, we use like `.bs-`... for example `.bs-checkbox` 
Do you think so? @djhvscf

**Comment by wenzhixin** [comment]
Great, thanks @djhvscf

**Pull request merged**